src/webapi/index.js

```diff
@@ -130,6 +130,23 @@ export default {
     return axios.get('/api/library/albums/' + albumId + '/tracks')
   },
 
+  library_genres () {
+    return axios.get('/api/library/genres')
+  },
+
+  library_genre (genre) {
+    if (genre) {
```

Comment:
Just drop the surrounding if statement. Passing genre is required for this function